cpp/cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -1850,12 +1849,6 @@ macro(build_gtest)
   set(GTEST_VENDORED TRUE)
   set(GTEST_CMAKE_CXX_FLAGS ${EP_CXX_FLAGS})
 
-  if(CMAKE_BUILD_TYPE MATCHES DEBUG)
-    set(CMAKE_GTEST_DEBUG_EXTENSION "d")
-  else()
-    set(CMAKE_GTEST_DEBUG_EXTENSION "")
-  endif()
-
```

Comment:
Could you also remove `${CMAKE_GTEST_DEBUG_EXTENSION}` from `_GTEST_LIBRARY_SUFFIX` and `_GTEST_RUNTIME_SUFFIX`?

Reply:
Done